### ISSUE TITLE
Wrong label: Wi-Fi (en0) - Ethernet 54 Mbps (similar to #155)

### ISSUE THREAD

**Issue by thehcma** [yujitach/MenuMeters#314]
On a MacBook Pro (Apple M3 Pro), very much like issue #155, MenuMeters is showing the bandwidth (incorrectly) as 54 Mbps.

![correct_bandwidth](https://github.com/yujitach/MenuMeters/assets/13666364/b9afd9f2-b6e9-4fb8-97e2-6a0bcc7dbd85)
![incorrect_bandwith](https://github.com/yujitach/MenuMeters/assets/13666364/de0f1bd9-e3c1-48ce-b572-06e45d30c303)

**Comment by thehcma** [comment]
fwiw, the MacOS version is: 14.3.1 (23D60)

**Comment by tomck** [comment]
I just migrated from a Mid 2012 MBP (MacBookPro9,2) on os 12.7.5 to a 2020 MBP (MacBookPro16,2) on os 14.5, and am having similar, but mine shows 0.0Kbps, whereas my old mac shows 450Mbps. I think this is just because this is kind of abandoned (no updates in 2 years), please see the readme for the project for references to other apps, I think Stats looks most promising. https://github.com/exelban/stats

**Comment by yujitach** [comment]
> I just migrated from a Mid 2012 MBP (MacBookPro9,2) on os 12.7.5 to a 2020 MBP (MacBookPro16,2) on os 14.5, and am having similar, but mine shows 0.0Kbps, whereas my old mac shows 450Mbps. I think this is just because this is kind of abandoned (no updates in 2 years), please see the readme for the project for references to other apps, I think Stats looks most promising. https://github.com/exelban/stats

Yeah I'm sorry for not having the time to maintain this app. Newer Mac's introduced new implementations for network interfaces, and new APIs are needed to get the information on them. Does Stats show the correct bps? If so I can copy their code to MenuMeters.

But in the end it's easier for you to just migrate to Stats.
